bin/tx-prover/README.md

```diff
@@ -1,84 +1,89 @@
 # Miden transaction prover
 
-A service for generating Miden transaction proofs on-demand. It is split in two binaries: worker and proxy.
+A service for generating Miden transaction proofs on-demand. It includes a CLI to control both workers and proxy for the Miden's remote transaction prover service, and a client, `RemoteTransactionProver` to interact with the service from a Rust codebase.
 
 The worker is a gRPC service that can receive transaction witnesses and returns the proof. It can only handle one request at a time and returns an error if is already in use.
 
 The proxy uses [Cloudflare's Pingora crate](https://crates.io/crates/pingora), which provides features to create a modular proxy. It is meant to handle multiple workers with a queue for each one. Further information about Pingora and it's features can be found in the [official GitHub repository](https://github.com/cloudflare/pingora).
 
 ## Installation
 
-Install the prover worker and proxy binaries for production using `cargo`:
+<!-- The following documentation is documented since the CLI is not on crates-io yet -->
```

Comment:
I think these can be taken out in order to have the instructions ready for when published to crates io (which should be fairly soon IIUC, cc @bobbinth)

Reply:
Do you mean to remove the comments?